src/Tickets/Commerce/Payments_Tab.php

```diff
@@ -57,41 +98,153 @@ public function register_tab() {
 
 		new \Tribe__Settings_Tab( static::$slug, esc_html__( 'Payments', 'event-tickets' ), $tab_settings );
 	}
-
-
+	
 	/**
-	 * Gets the top level settings for Tickets Commerce.
+	 * Returns the settings item for the section menu at the top of the Payments settings tab.
 	 *
-	 * @since 5.2.0
+	 * @since TBD
 	 *
 	 * @return array[]
 	 */
-	public function get_top_level_settings() {
-
-		$plus_link    = sprintf(
-			'<a href="https://evnt.is/19zl" target="_blank" rel="noopener noreferrer">%s</a>',
-			esc_html__( 'Event Tickets Plus', 'event-tickets' )
-		);
-		$plus_link_2  = sprintf(
-			'<a href="https://evnt.is/19zl" target="_blank" rel="noopener noreferrer">%s</a>',
-			esc_html__( 'Check it out!', 'event-tickets' )
-		);
-		$plus_message = sprintf(
-		// Translators: %1$s: The Event Tickets Plus link, %2$s: The word "ticket" in lowercase, %3$s: The "Check it out!" link.
-			esc_html_x( 'Tickets Commerce is a light implementation of a commerce gateway using PayPal and simplified stock handling. If you need more advanced features, take a look at %1$s. In addition to integrating with your favorite ecommerce provider, Event Tickets Plus includes options to collect custom information for attendees, check attendees in via QR codes, and share stock between %2$s. %3$s', 'about Tickets Commerce', 'event-tickets' ),
-			$plus_link,
-			esc_html( tribe_get_ticket_label_singular_lowercase( 'tickets_fields_settings_about_tribe_commerce' ) ),
-			$plus_link_2
-		);
-
-		$is_tickets_commerce_enabled = tec_tickets_commerce_is_enabled();
-
+	public function get_section_menu() {
+		
+		$selected_section = tribe_get_request_var( self::$key_current_section_get_var );
+		
+		$sections = [
+			[
+				'classes' => [],
+				'url' => Tribe__Settings::instance()->get_url( [ 'tab' => 'payments' ] ),
+				'text' => __( 'Tickets Commerce', 'event-tickets' )
+			]
+		];
+		if ( empty( $selected_section ) ) {
+			$sections[0]['classes'][] = 'active';
+		}
+		
+		$gateways = tribe( Manager::class )->get_gateways();
+		
+		foreach ( $gateways as $gateway_key => $gateway ) {
+			if ( ! $gateway::should_show() ) {
+				continue;
+			}
+			$new_section = [
+				'classes' => [],
+				'url' => Tribe__Settings::instance()->get_url( [ 'tab' => 'payments', self::$key_current_section_get_var => $gateway->get_key() ] ),
+				'text' => $gateway->get_label(),
+			];
+			if ( $selected_section === $gateway->get_key() ) {
+				$new_section['classes'][] = 'active';
+			}
+			$sections[] = $new_section;
+		}
+		
+		$admin_views = tribe( Tribe__Tickets__Admin__Views::class );
+		$admin_views->set_template_folder( 'src/admin-views/settings/tickets-commerce/section' );
```

Comment:
Same problem as the other comment I made, `Tribe__Tickets__Admin__Views` is a singleton. you cannot change that folder.